traveler-ios-ui/TravelerKitUI/ViewControllers/BookingQuestionsViewController.swift

```diff
@@ -35,14 +36,22 @@ class BookingQuestionsViewController: UIViewController {
 
     @objc
     func keyboardWillChangeFrame(_ note: Notification) {
+        var bottomInset:CGFloat = 0.0
+
         guard let keyboardFrame = note.userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect else {
             return
         }
 
         let keyboardFrameInView = view.convert(keyboardFrame, from: nil)
-        let bottomInset = view.safeAreaLayoutGuide.layoutFrame.intersection(keyboardFrameInView).height
+
+        if let safeArea = originalSafeArea {
```

Comment:
This strategy doesnt account for changes to the VCs frame. I understand that this is not an issue here in this case, but if for whatever reason sizes change (Remember, I want to be able to use any VC independently) this logic would break.

I think there is a better way to do this without using instance variables to store a value at a given time (which may change)